storage/innobase/xtrabackup/test/t/bug1192834-rollback.sh

```diff
@@ -0,0 +1,44 @@
+##########################################################################
+# Bug 1192834: Crash during apply with index compaction enabled          #
+##########################################################################
+
+. inc/common.sh
+
+start_server --innodb_file_per_table
+load_dbase_schema sakila
+load_dbase_data sakila
+
+function start_uncomitted_transaction()
+{
+    run_cmd $MYSQL $MYSQL_ARGS sakila <<EOF
+START TRANSACTION;
+DELETE FROM payment;
+SELECT SLEEP(10000);
+EOF
+}
+
+start_uncomitted_transaction &
+job_master=$!
+
+sleep 2
```

Comment:
I want DELETE to actually delete some rows before backup starts, I could grep SHOW PROCESSLIST output, but it would be overkill in this case.